bot/cogs/verification.py

```diff
@@ -155,6 +161,20 @@ def bot_check(ctx: Context) -> bool:
         else:
             return True
 
+    @tasks.loop(hours=1.0)
+    async def periodic_ping(self) -> None:
+        """Post a recap message every week with an @everyone."""
+        message = await self.bot.get_channel(Channels.verification).history(limit=1).flatten()  # check last message
```

Comment:
I'm wondering if it might be better to be more specific about the message to check. There is a small chance that the last message will be the bot saying "`@someuser` Please type `!accept` to verify that you have ...", which the bot will do whenever someone types something other than `!accept`. This message stays for about 10 or 15 seconds before being deleted. in other words, there's a chance this will fail even when it shouldn't.

Of course, the price for such a failure is very low, it just means that we'll wait a little longer before posting this, so I don't know if it's worth fixing, but a slightly more clever method for ensuring that we're checking the correct message would prevent it.